nest/neststartup.cpp

```diff
@@ -127,27 +127,15 @@ neststartup( int* argc, char*** argv, SLIInterpreter& engine, std::string module
  * The following section concerns shared user modules and is thus only
  * included if we built with libtool and libltdl.
  *
- * On BlueGene, we need to link user modules statically, but for convenience
+ * On some machines, we need to link user modules statically, but for convenience
  * they still register themselves with the DyamicLoadModule during static
- * initialization. On BlueGene, we then need to prevent that the modules are
- * loaded a second time, therefore the guard below. At the same time, we
- * need to create the DynamicLoaderModule, since the compiler might otherwise
- * optimize DynamicLoaderModule::registerLinkedModule() away.
+ * initialization. At the same time, we need to create the DynamicLoaderModule,
+ * since the compiler might otherwise optimize DynamicLoaderModule::registerLinkedModule() away.
  */
 #ifdef HAVE_LIBLTDL
   // dynamic loader module for managing linked and dynamically loaded extension
   // modules
   nest::DynamicLoaderModule* pDynLoader = new nest::DynamicLoaderModule( engine );
-
-// initialize all modules that were linked into at compile time
-// these modules have registered via calling DynamicLoader::registerLinkedModule
-// from their constructor
-#ifndef IS_BLUEGENE
```

Comment:
Since this is an `#ifndef`, I believe we need to just remove the guard but keep the code.

Reply:
Thanks for spotting this, I was a bit quick when removing things. Fixed now.